zerver/models.py

```diff
@@ -1040,6 +1040,10 @@ def allowed_bot_types(self) -> List[int]:
     def emojiset_choices() -> Dict[str, str]:
         return OrderedDict((emojiset[0], emojiset[1]) for emojiset in UserProfile.EMOJISET_CHOICES)
 
+    @staticmethod
+    def emojiset_choices_sort() -> List[str]:
+        return [emojiset[0] for emojiset in UserProfile.EMOJISET_CHOICES]
```

Comment:
Let's not introduce a duplicate function and field to fix this bug.  We should fix the existing `emojiset_choices` to what what we need.  I think a list of dicts is a totally reasonable new format for it, replacing the `OrderedDict` we have currently.